docs/deploy.md

```diff
@@ -179,16 +185,22 @@ bw_db
 ```
 pixi run metadata_sra
 ```
-> to build a smaller dataset for testing, run `pixi run metadata_sra_test`
-
+```{note}
+to build a smaller dataset for testing, run `pixi run metadata_sra_test`
```

Comment:
```suggestion
to build a smaller dataset for testing, run `pixi run metadata_sra --build-test-db`
```
did you want to switch to this as you suggested via slack? Could also delete that line from `pyproject.toml`